tests/AC_model_manager/test_model_download.py

```diff
@@ -0,0 +1,361 @@
+"""Test the queued download facility"""
+
+import tempfile
+import time
+from pathlib import Path
+
+import requests
+from requests import HTTPError
+from requests_testadapter import TestAdapter
```

Comment:
I don't love that we're bringing in an unmaintained dependency (hasn't been updated in 10 years, and the Github project has been archived). I think the same thing could be achieved relatively easily with [`unittest.mock`](https://docs.python.org/3/library/unittest.mock.html).

Reply:
I dunno. It was trivial to get requests_testadapter working, but with mock I'm going to have to implement stuff like  HTTP header handling. 

requests_testadapter is a trivial thing - just 86 non-blank lines. It's still working after 10 years and if it gets broken it will be super-easy to reimplement.